[Cases] Deprecated endpoints (#1494)
elastic/security-docs: docs/cases/api/cases-api/cases-api-get-case-activity.asciidoc

NOTE: Console supports only Elasticsearch APIs. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to {kibana-ref}/console-kibana.html[Run {es} API requests].

> WARNING: This endpoint is deprecated and will be removed in future releases.

Reviewer: If we can't offer a concrete statement about when the endpoint will be removed, can we state the last version it's supported on?

Author: @nastasha-solomon Ok! Any suggestions? Thanks!

Reviewer: If there are no legal constraints or other limitations to consider, maybe something like, "this endpoint is not available after version 8.1" or "this endpoint is available until version 8.1".

Author: > If there are no legal constraints or other limitations to consider
I am not aware of any. Do you have something in mind?

Reviewer: Gotcha. Just recommending a minor edit then! Thanks again for your work on this!

> WARNING: This endpoint is deprecated and it will not be supported in future releases.

Reviewer 2: What do you think about:

```suggestion
WARNING: This endpoint is deprecated and will be removed in future releases.
```
